python/cudf/cudf/tests/test_series.py

```diff
@@ -395,7 +395,9 @@ def test_series_describe_numeric(dtype):
     actual = gs.describe()
     expected = ps.describe()
 
-    assert_eq(expected, actual)
+    # Have to set check_dtype=False because:
+    # https://github.com/pandas-dev/pandas/issues/48340
```

Comment:
I think we'll fix this on the pandas side: https://github.com/pandas-dev/pandas/pull/48473